openhtf/plugs/__init__.py

```diff
@@ -405,7 +414,8 @@ def initialize_plugs(self, plug_types=None):
         self.tear_down_plugs()
         raise
       self.update_plug(plug_type, plug_instance)
-    self._initialize_rpc_server()
+    if not partial:
+      self._initialize_rpc_server()
```

Comment:
I thought the RPC server is the whole point of this? Otherwise, how will we get plugs in test start to communicate with the frontend?